src/PKSim.Core/Model/Extensions/DataColumnExtensions.cs

```diff
@@ -17,13 +17,13 @@ public static bool BelongsTo(this DataColumn dataColumn, Simulation simulation)
          if (simulation == null || !simulation.HasResults)
             return false;
 
+         if (simulation.IsAnImplementationOf<PopulationSimulation>())
+            return true;
+
          if (dataColumn.DataInfo.Origin != ColumnOrigins.Calculation)
             return false;
 
-         if (individualSimulationContainsColumn(dataColumn, simulation))
-            return true;
-
-         return simulation.IsAnImplementationOf<PopulationSimulation>();
+         return individualSimulationContainsColumn(dataColumn, simulation);
```

Comment:
Don't quite understand this change. Seems to be equivalent to me?

Reply:
For what I have seen in the code, when we receive a population simulation the DataInfo.Origin is not properly set. My thought was that for the case of populations it is not meaningful (or at least in this case) but as we discussed, it makes sense to see why it has not been set. So the change is indeed very similar but the only difference is that for population simulations it will not check the origin at all. Before it would return false always. I will still check why the origin is not been set.